config/config.exs

```diff
@@ -412,6 +412,10 @@ config :screens, :screens_by_alert,
   screens_last_updated_ttl_seconds: 3600,
   screens_ttl_seconds: 40
 
+config :screens, Screens.ScreenApiResponseCache,
+  # => 1 hr
+  gc_interval: 3_600_000
```

Comment:
Formatting came out a little weird here with the comment. As an alternative, can we do `:timer.hours(1)`?